exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,14 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
+    return "failed";
+  }
+  else if (grade >= 50) {
+    return "passes"
+  }
+}
```

Comment:
Good job Somayah. If you only have two options, you do not need to use else if.
in this scenario, the first option was less than 50, that means the other one will definetly by >=50.

you can use
 return "passes"
 without elseif statement, and it will still work